event-log-encoder/src/main/scala/com/ubirch/encoder/process/EncoderExecutor.scala

```diff
@@ -111,113 +111,66 @@ class EncoderExecutor @Inject() (
   val UPP: PartialFunction[EncoderPipeData, Option[EventLog]] = {
 
     case encoderPipeData @ EncoderPipeData(_, Vector(jv)) if Try(EncoderJsonSupport.FromJson[MessageEnvelope](jv).get).isSuccess =>
-
-      (for {
-
-        messageEnvelope <- Try(EncoderJsonSupport.FromJson[MessageEnvelope](jv).get)
-
-        _ = encodingsCounter.counter.labels(metricsSubNamespace, Values.UPP_CATEGORY).inc()
-
-        customerId <- Try((messageEnvelope.context \\ CUSTOMER_ID_FIELD).extract[String])
-          .filter(_.nonEmpty)
-          .recoverWith {
-            case _ => throw EventLogFromConsumerRecordException("No CustomerId found", encoderPipeData)
-          }
-
-        payload <- Try(EncoderJsonSupport.ToJson[ProtocolMessage](messageEnvelope.ubirchPacket).get)
-
-        eventLog <- Try(messageEnvelope)
-          .map(_.ubirchPacket)
-          .filter(_.getHint == 0)
-          .map { _ =>
-
-            val payloadJsNode = Option(messageEnvelope)
```

Comment:
this part creating each field were replaced with helper methods in the EncodeHelper object.

Reply:
you use the wrong topic ubirch-svalbard-evt-log-json, delete/disable/enable UPPs should be send to a new topic

Reply:
I like the EncodeHelper very much. It makes it much more readable. Couldn't the customerId retrieval also get moved to the helper?

Reply:
and the getCategory method maybe as well?

Reply:
> you use the wrong topic ubirch-svalbard-evt-log-json, delete/disable/enable UPPs should be send to a new topic

yes. I'll change it after I add new topic on the helm chart and it's created on dev.

Reply:
> I like the EncodeHelper very much. It makes it much more readable. Couldn't the customerId retrieval also get moved to the helper?

ah, yes. I'll move it. the getCategory method as well. thanks